### DISCUSSION TITLE
Images nextjs github actions

### DISCUSSION THREAD

**Discussion by kwame-Owusu** [community/community#51139] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi everyone, I deployed my nextjs app, which is a portfolio website about me, but there is are no images appearing,  but eveyimage is in the repository, so i dont know how to fix this, i need help, thanks in advance.

**Comment by Anmol-Baranwal** [comment]
share the repo link

**Reply by kwame-Owusu** [reply]
I have found the solution, Instead of using github actions I have deployed the app on vercel, and there are no problems, but ill share the link so you can see the code, and maybe suggest if there is anything I did wrong: https://github.com/kwame-Owusu/Kwame

**Comment by kris621294** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> Hi everyone, I deployed my nextjs app, which is a portfolio website about me, but there is are no images appearing,  but eveyimage is in the repository, so i dont know how to fix this, i need help, thanks in advance.

**Comment by Anmol-Baranwal** [comment, marked as answer]
> My apologies, it appears that our conversation got buried in the midst of other discussions. 

I checked the repository you shared, and it seems that the path of images is also correct, so I don't really know what's wrong :)

However, it's worth noting that GitHub Pages has some limitations and quirks when it comes to deploying Next.js apps. You may encounter issues such as blank pages or missing assets, especially if you are using features such as server-side rendering or dynamic imports.

In contrast, Vercel is a platform that is specifically designed to host and deploy Next.js apps, so you are less likely to encounter these issues.

> Please inform me, if you get a reason for the issue.

**Reply by kwame-Owusu** [reply]
yeah eveything you said is spot on, after reviewing my code, there seems to be nothing wrong with it, it just has to do with github actions, and it is running flawlessly on vercel.
